### PR TITLE
Add support for externaladmissionhookconfigurations and initializerco…

### PR DESCRIPTION
**Pull request by shiywang**
Fixes https://github.com/kubernetes/kubernetes/issues/48010
/assign @liggitt 

> It's a missing printer. There are two issues:
> 1. the printer is missing
> 2. kubectl should fall back to a generic printer for types without registered printers

this pr now addressed 1, for 2, still wip

### CODE REVIEW

**Comment by shiywang** [comment]
@liggitt @smarterclayton will rebase after https://github.com/kubernetes/kubernetes/pull/48033 get merged

File: pkg/printers/internalversion/printers.go [hunk 1]
```diff
@@ -201,6 +204,10 @@ func AddHandlers(h printers.PrintHandler) {
 	h.Handler(statusColumns, nil, printStatus)
 	h.Handler(controllerRevisionColumns, nil, printControllerRevision)
 	h.Handler(controllerRevisionColumns, nil, printControllerRevisionList)
+	h.Handler(initializerConfigurationColumns, nil, printInitializerConfiguration)
```
Comment (smarterclayton) [inline pkg/printers/internalversion/printers.go]: Please add a new style printer (table) rather than old style.  See other examples in this file

**Review by smarterclayton — CHANGES_REQUESTED** [review]

File: pkg/printers/internalversion/printers.go (hunk 1 above)
Comment (shiywang) [inline pkg/printers/internalversion/printers.go]: for now, `initializerConfigurationColumns` only print NAME and AGE, isn't it already covered by  AddDefaultHandlers ?

**Pull request closed without merging**